README.md

```diff
@@ -6,27 +6,45 @@ The derived reactors customize this base class for particular MuJoCo model files
 
 ## Prerequisites
 
-MuJoCo depends on [GLFW](https://www.glfw.org), a graphics library that you must install.  On macOS:
+
+### Linux (Ubuntu)
+
+1. Install GLFW, a graphics library used by MuJoCo
 
 ```sh
-brew install glfw
+apt install libglfw3-dev
 ```
 
-MuJoCo itself seems to be best installed from source.  The following worked for me on macOS:
+2. Download pre-built Mujoco v3.2.6 binaries from [here](https://github.com/google-deepmind/mujoco/releases/tag/3.2.6).
+```sh
+wget wget https://github.com/google-deepmind/mujoco/releases/download/3.2.6/mujoco-3.2.6-linux-x86_64.tar.gz
+```
 
+3. Install to `/opt` 
 ```sh
-git clone git@github.com:google-deepmind/mujoco.git
-cd mujoco
-mkdir build
-cd build
-cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local
-cmake --build .
-sudo cmake --install .
+sudo mkdir -p /opt/mujoco-3.2.6-linux-x86_64/
```

Comment:
I am installing to `/opt` which on Linux is a convention for manually installed packages